server/graphql-schema.js

```diff
@@ -98,7 +98,7 @@ const graphqlSchema = gql`
 
     type TestPlanVersion {
         id: ID!
-        title: String!
+        title: String
```

Comment:
Interesting! 

Thinking from a business perspective, do you think it should remain required (and will use the directory name if an explicit name is not provided)?

Reply:
Well I think there would still be need for the application to know the `title` different from the `directory` here so I don't view it as a 'backup' in this instance.
However, I do think my changing of the schema here isn't valid. I believe I did this when nested queries were failing when pulling `TestPlanVersion`s with `null` titles. This should certainly be managed from the resolver.